src/bp-groups/classes/class-bp-groups-member.php

```diff
@@ -294,6 +294,9 @@ public function save() {
 		if ( ! empty( $this->id ) ) {
 			$sql = $wpdb->prepare( "UPDATE {$bp->groups->table_name_members} SET inviter_id = %d, is_admin = %d, is_mod = %d, is_banned = %d, user_title = %s, is_confirmed = %d, comments = %s, invite_sent = %d WHERE id = %d", $this->inviter_id, $this->is_admin, $this->is_mod, $this->is_banned, $this->user_title, $this->is_confirmed, $this->comments, $this->invite_sent, $this->id );
 		} else {
+			if ( ! is_admin() ) {
```

Comment:
@milanhirapra Can you explain how this condition will exclude invited organizers with pending status?

Reply:
@dipskakadiya The administrator/admin user can insert the user/member into the group from the backend. So, this condition is true if the request is from the admin side otherwise false.